### DISCUSSION TITLE
Windows layout

### DISCUSSION THREAD

**Discussion by Sanoilp** [rustdesk/rustdesk#9083] [General]
Used for monitoring multiple remote computers, it can neatly arrange windows as needed.
If it can be integrated into a single window, where small windows can monitor in real-time, and clicking on a small window can enlarge that remote computer, that would be even better!
<a href="https://ibb.co/dgLK2kX"><img src="https://i.ibb.co/5Fnc8sC/8.png" alt="8" border="0"></a>

**Comment by Sanoilp** [comment]
How about adding this feature?

**Comment by EdoCocc** [comment]
Hi! I'd like to bump this discussion and add my strong support for this feature.

A 'CCTV-style' grid view or dashboard would be incredibly valuable for my workflow. Specifically, I would use it to monitor my manufacturing environment with a lot of machines.

Having a low-framerate, multi-screen overview that allows clicking into a specific machine for full control would elevate RustDesk to the level of enterprise management tools.

Are there any technical roadmaps that might include this, or is there a specific technical hurdle preventing it currently?
